### DISCUSSION TITLE
The `identify.exe -ping` command fails when supplied with globbing pattern

### DISCUSSION THREAD

**Discussion by yansklyarenko** [dlemstra/Magick.NET#691] [Help] [answered]
I'm observing a strange behavior of the `identify.exe` utility (Windows 10). 
Let's say I have a single JPG file in a directory. If that file has a usual name (for example, `image.jpg`), the following command works just fine:
`C:\ProgramData\chocolatey\bin\identify.exe -ping *.jpg`
and produces expected output:
`image.jpg JPEG 318x313 318x313+0+0 8-bit sRGB 12853B 0.000u 0:00.000`

However, as soon as the file name starts with `.` - the same command fails with the following error:
`identify.exe: UnableToOpenBlob '*.jpg': Invalid argument @ error/blob.c/OpenBlob/3497.`

It's interesting that if you supply the exact file name to the command, it works just fine:
`C:\ProgramData\chocolatey\bin\identify.exe -ping .starts_with_dot.jpg`
outputs:
`.starts_with_dot.jpg JPEG 318x313 318x313+0+0 8-bit sRGB 12853B 0.000u 0:00.000`

Is it expected? Is it possible to work this around by providing some parameters to the `identify.exe`? 

Thanks in advance!

**Comment by dlemstra** [comment, marked as answer]
Could you move the discussion to the ImageMagick repository: https://github.com/ImageMagick/ImageMagick/discussions?
